activiti-core/activiti-engine/src/main/java/org/activiti/engine/impl/util/ProcessInstanceHelper.java

```diff
@@ -129,11 +129,13 @@ else if (bpmnModel.containsMessageId(messageRef)) {
       throw new ActivitiException("No message start event found for process definition " + processDefinition.getId() + " and message name " + messageName);
     }
 
-    // Map message payload variables before creating process instance
-    Map<String, Object> processVariables = commandContext.getProcessEngineConfiguration()
-                                                         .getEventSubscriptionPayloadMappingProvider()
-                                                         .apply(messageVariables,
-                                                                eventSubscription);
+//    // Map message payload variables before creating process instance
+//    Map<String, Object> processVariables = commandContext.getProcessEngineConfiguration()
```

Comment:
If this code is not necessary anymore it should be completely removed. I guess that this is removing support for variable mappings for messages when declared inside the XML itself, but maybe it's not a big deal, as all the variable mappings are done using the extension files, right now. @igdianov do you any concern about that?

Reply:
Done